### ISSUE TITLE
Warzone Tiers

### ISSUE THREAD

**Issue by ColonCrusader** [SmashMC-Development/Suggestions#1022]
So i would really enjoy seeing something along the lines of what smogon has in place for showdown, like tiers for Uber, OU, UU, etc., and i would like to see this in warzone. im aware it would be a lot of work, but i think it would be interesting to see tiers to wz, like a uber tier, where any Pokémon can be brought, but for the ou tier, you can only bring Pokémon in the ou tier or lower, same for uu etc. a way to balance this out that i think could work pretty well, is have only the Pokémon that spawn in that tier or lower can spawn. So for example a kyogre cant spawn in the uu tier, but it works the other way, like a charizard can spawn in uber tier. I think this would make it more enjoyable for people that want to try new mons that they enjoy, that aren't as viable compared to things like kyogre, and you have to go up against similar teams. I feel it would help balance WZ and allow people who want to use Pokémon in lower tiers they enjoy to not have to go up against ubers etc.

This would apply to all servers.

In my personal opinion i think it would be nice to see larger maps for the higher tiers with smaller maps for the lower tiers. I don't mean like 64x64 maps, but i don't think the maps for lower tier WZ need to be as fancy and impressive as the higher tiers

**Comment by BrunchBee** [comment]
With that being said I don't think we would need an NU, PU, or LC warzone 😅at a certain point you're just wasting money even more than you already are when you go to wz

**Comment by Happi1223** [comment]
i mean if you want this there could be like a timeframe where like certain tiers are only allowed for a certain amount of time to change up the vibe. say for like 3-5 days only UU can be used then after OU, Uber and then AG and it would just be a rotation now it would probably be a pain so it wouldnt be done but that would be the idea if you wanted different tiers. Im pretty sure this is what you were getting at but the stuff spawning is not really important bc you go in there to catch things and fight and if only UU or whatever spawn in that cycle wz will just be dead cuz no one will want anything there and then when uber tier is tehre itll be alive bc everyone will want everything. I dont think this will happen but that would be the idea of what could

**Comment by ColonCrusader** [comment]
> With that being said I don't think we would need an NU, PU, or LC warzone 😅at a certain point you're just wasting money even more than you already are when you go to wz

Fair point, i think if it went down to uu that would be good enough

**Comment by ColonCrusader** [comment]
> i mean if you want this there could be like a timeframe where like certain tiers are only allowed for a certain amount of time to change up the vibe. say for like 3-5 days only UU can be used then after OU, Uber and then AG and it would just be a rotation now it would probably be a pain so it wouldnt be done but that would be the idea if you wanted different tiers. Im pretty sure this is what you were getting at but the stuff spawning is not really important bc you go in there to catch things and fight and if only UU or whatever spawn in that cycle wz will just be dead cuz no one will want anything there and then when uber tier is tehre itll be alive bc everyone will want everything. I dont think this will happen but that would be the idea of what could

interesting idea, i think it could make things interesting

**Comment by manlybro2002** [comment]
sounds good as a separate content to wz, its fun havin different teams without stickin to a specific format in wz.

**Comment by oLimooo** [comment]
I disagree with this suggestion about tiers in warzone. When warzone first was added into Smash it was “AG”, meaning you can take any pokemon into warzone. Giving teirs in warzone would be a huge mess imo. In addition, if players would like to challenge themselves by using a uu team in warzone, they are welcome to, it’s AG.  Also take into consideration that AG, OU, UU all have different  restrictions meaning different rules per tier and different explanations. Apart of that, it would be a mess to give a explain on how warzone works to new non competitive players. How warzone was past reset I say it’s perfect from my POV.

**Comment by Indorlan** [comment]
Problem is its not rly AG as some things like moody and evasion tactics are banned, but things like Kyogre and Zacian are not.

**Comment by oLimooo** [comment]
You’re right warzone does add some restrictions into warzone, however there is no restrictions on which pokemon a player can use. For example, Moody is banned because it creates a “unfair” advantage (IMO). using spore+Focus sash+Protect then using baton pass makes a battle kinda unfair. When warzone was added it was full AG, all we saw was a bunch of op strategy that players abused.  Now warzone has some restrictions to create a type of “AG” that is balanced to actually create the competitive in warzone. Nevertheless the warzone now is better than any other previous one (based on restrictions).
